91.md

```diff
@@ -0,0 +1,48 @@
+NIP-91
+======
+
+IoT sensors and intents
+-----------------------
+
+`draft` `optional` `author:BlackCoffee` `author:arcbtc` `author:fiatjaf`
+
+This NIP standardizes communication of public sensor and intent events in order to create an open, pluggable and interoperable standard for "Internet of Things" devices to communicate.
+
+The standard event format follows the following template:
+
+```js
+{
+  "kind": <a number from the table below>,
+  "tags": [
+    ["device", <optional device id>],
+    ["alt", <optional human-readable translation of the event>],
+  ],
+  "content": "<data according to the format in the table below>"
+  ...
+}
+```
+
+Ideally each device would have its own public key, which allows for easy command-and-control and comes with built-in trus guarantees (i.e. you know you're getting the data from the correct device if you know its key and can check the signature), but the `device` tag exists for when that is too cumbersome and more than one device is using the same key.
```

Comment:
trust guarantees?

Reply:
What I meant was that you can know for sure who is publishing the data given the signature, but it's definitely confusing. Please reword.

Reply:
> What I meant was that you can know for sure who is publishing the data given the signature, but it's definitely confusing. Please reword.

I got it and the previous comment is about typo haha. 
suggestions: 
"and ensures data authenticity with build-in properties. "

Reply:
Sorry, sir, I'm not an English speaker (and I still don't see the typo).

Reply:
"built-in trus guarantees" is missing a `t` in trust.